students/brandon_nguyen/session02/fizzBuzz.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+#Fizz Buzz problem - Week 2 Excercise 1
+#Student: Brandon Nguyen - Au2018
+for i in range(100):
+    i +=1
```

Comment:
Hmm... Do you get the right results? I doubt so, because this line will make the loop skip every other number, which is not right. In other words, because of this line, your `i` will be 1, 3, 5, ..., 99, not 1, 2, 3, 4, 5, ..., 100. Note that `range(100)` already allows the `for` loop to iterate over 0, 1, 2, 3, 4, ..., 99 (100 not included). So you shouldn't have this `i += 1` statement inside your loop. Another thing is that if you want to iterate over from 1 to 100, you need to use `range(1, 101)` to specify the starting value of 1, and the last value is always NOT included.